frontend/src/scenes/session-recordings/player/sidebar/PlayerSidebarOverviewOtherWatchers.tsx

```diff
@@ -17,19 +21,53 @@ function OtherWatchersLoading(): JSX.Element {
 }
 
 function OtherWatchersDisplay({ metadata }: { metadata?: SessionRecordingType }): JSX.Element | null {
+    const { user: currentUser } = useValues(userLogic)
+    const [isExpanded, setIsExpanded] = useState(false)
+
     if (!metadata?.viewers) {
-        // to keep TS happy
         return null
     }
 
-    const count = metadata.viewers.length
+    // Filter out the current user from the viewers list
+    const otherViewers = metadata.viewers.filter((viewer) => viewer !== currentUser?.email)
+    const count = otherViewers.length
+
+    if (count === 0) {
+        return null
+    }
+
     const varyingText = count > 1 ? 'users have' : 'user has'
+
+    const toggleExpanded = (): void => {
+        setIsExpanded(!isExpanded)
+    }
+
     return (
-        <div className="flex flex-row deprecated-space-x-2 items-center justify-center px-2 py-1">
-            <ProfileBubbles people={metadata.viewers.map((v) => ({ email: v }))} />
-            <span>
-                {count} other {varyingText} watched this recording.
-            </span>
+        <div className="flex flex-col gap-2 px-2 py-1">
+            <div className="flex flex-row deprecated-space-x-2 items-center justify-center">
+                <ProfileBubbles people={otherViewers.map((v) => ({ email: v }))} />
```

Comment:
ah, it is a shared component but we have this additional text anyway

i think we can ignore that it's shared for now